src/types/interfaces.ts

```diff
@@ -113,7 +114,14 @@ export interface ISdkConfig {
    * }
    * ```
    */
-  logger?: ILogger;
+  logger?: Logger;
```

Comment:
This is the interface that consumers use to construct the SDK. This param is for them to pass in their own logger which most likely will not be an instance of our gc-logger. I think this type needs to be the `ILogger` interface exported from gc-logger. 

Also, we need to make sure we export `ILogger` from the SDK's `index.ts` file to make sure consumers can type their applications without having to import directly from the gc-logger

Reply:
Yep, thanks. This one was accidental.